src/components/Section/IconSection.js

```diff
@@ -0,0 +1,40 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {View} from 'react-native';
+import Icon from '@components/Icon';
+import useThemeStyles from '@hooks/useThemeStyles';
```

Comment:
Does this have some dependency which is not migrated to TS or can we make this in TS now?